tests/sauceDemo_tests.js

```diff
@@ -0,0 +1,107 @@
+
```

Comment:
Delete empty rows

Reply:
Deleted row 1,2 and 22.
Two empty rows set between each portion of code.